MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -405,10 +405,16 @@ private async Task<PlaybackInfoResponse> GetPlaybackInfo(Guid id, Guid userId, s
                     user.Policy.EnableAudioPlaybackTranscoding);
             }
 
+            // Beginning of Playback Determination: Attempt DirectPlay first
             if (mediaSource.SupportsDirectPlay)
             {
-                if (mediaSource.IsRemote && forceDirectPlayRemoteMediaSource)
+                if (mediaSource.IsRemote && forceDirectPlayRemoteMediaSource && user.Policy.ForceRemoteSourceTranscoding)
                 {
+                    mediaSource.SupportsDirectPlay = false;
+                }
+                else if (mediaSource.IsRemote && user.Policy.ForceRemoteSourceTranscoding)
+                {
+                    mediaSource.SupportsDirectPlay = false;
                 }
```

Comment:
so if I condense repeated cases, it would look like
```c#
if (mediaSource.IsRemote && user.Policy.ForceRemoteSourceTranscoding)
{
    mediaSource.SupportsDirectPlay = false;
}
```

note that `forceDirectPlayRemoteMediaSource` is now gone, which looks strange

Reply:
I can't find the code which handles `forceDirectPlayRemoteMediaSource` 

I found this but it doesn't have information for forced directplay of remote sources.
https://github.com/jellyfin/jellyfin-web/blob/86bf6ab8d2999398b3d1775f5f20ba659ef3fd0d/src/components/playback/playbackmanager.js#L1772

Reply:
@JustAMan I couldn't find anything for `forceDirectPlayRemoteMediaSource` and feel like it might be a dangling source. Could you verify before I rip it out?

Reply:
@Artiume It looks to me like `forceDirectPlayRemoteMediaSource` is always `true`.

Reply:
@cvium Yeah, I honestly want to remove forceDirectPlayRemoteMediaSource altogether. It's only used in this file and it doesn't actually do anything.

Reply:
Did you remove it?

Reply:
@dkanada it is still located at L330, L340 and 360.

I didnt touch those portions of the code, it could probably be removed safely from L340. not sure about 330, 360